plexpy/pmsconnect.py

```diff
@@ -692,11 +697,225 @@ def get_metadata_children_details(self, rating_key=''):
         output = {'metadata': metadata_list}
         return output
 
-    """
-    Return processed and validated session list.
 
-    Output: array
-    """
+    def get_watched_status(self, sort=None, sections='all', all_params=False, get_file_size=True, 
+                           exclude_path=None, watched_older_then=None, hide_watched=0, ignore_section='',
+                           *args, **kwargs):
+
+        """
+        Returns a list of all unwatched shows
+
+        named args: Used for enabled and disabling sorting/filtering
+        kwargs: Used for filtering inside the dicts. Adding type="movie" will only returns movies
+
+
+        Output: List for dicts
+
+        kwargs: Used for filtering inside the dicts. Adding type="movie" will only list movies
+
+
+        Output: List for dicts
+
+        # Adding all_params=1 Makes the call insane slow.
+        """
+        # Add a cache?
+
+        use_watched_older_then_sort = True
+        if watched_older_then is None:
+            use_watched_older_then_sort = False
+            watched_older_then = time.time()
+        else:
+            watched_older_then = int(watched_older_then)
+
+        if plexpy.CONFIG.PMS_URL:
+            url_parsed = urlparse(plexpy.CONFIG.PMS_URL)
+            hostname = url_parsed.hostname
+            port = url_parsed.port
+            self.protocol = url_parsed.scheme
+        else:
+            hostname = plexpy.CONFIG.PMS_IP
+            port = plexpy.CONFIG.PMS_PORT
+            self.protocol = 'http'
+
+        b_url = '%s://%s:%s' % (self.protocol, hostname, port) 
+
+        h = {'Accept': 'application/json',
+             'X-Plex-Token': plexpy.CONFIG.PMS_TOKEN
+            }
+
+        hide_watched = 'unwatched' if hide_watched else 'all'
```

Comment:
I believe `/library/sections/<key>/unwatched` only lists unwatched for the logged in user (i.e. yourself)?

Something could be watched by someone else, and we don't want it to show up as unwatched.